$NetBSD: patch-af,v 1.2 2009/04/12 01:03:24 sno Exp $

Fix culled from following references in
http://rt.cpan.org/Public/Bug/Display.html?id=40698
Expected output is now only present when VERBINST=1.

--- t/basic.t.orig	2009-02-20 00:30:36.000000000 +0000
+++ t/basic.t	2009-04-12 00:52:42.000000000 +0000
@@ -130,7 +130,7 @@
     diag $test_out;
 
 
-my $install_out = run("$make install");
+my $install_out = run("$make install VERBINST=1");
 is( $?, 0, 'install' ) || diag $install_out;
 like( $install_out, qr/^Installing /m );
 
@@ -155,7 +155,7 @@
 SKIP: {
     skip 'VMS install targets do not preserve $(PREFIX)', 9 if $Is_VMS;
 
-    $install_out = run("$make install PREFIX=elsewhere");
+    $install_out = run("$make install VERBINST=1 PREFIX=elsewhere");
     is( $?, 0, 'install with PREFIX override' ) || diag $install_out;
     like( $install_out, qr/^Installing /m );
 
@@ -174,7 +174,7 @@
 SKIP: {
     skip 'VMS install targets do not preserve $(DESTDIR)', 11 if $Is_VMS;
 
-    $install_out = run("$make install PREFIX= DESTDIR=other");
+    $install_out = run("$make install VERBINST=1 PREFIX= DESTDIR=other");
     is( $?, 0, 'install with DESTDIR' ) || 
         diag $install_out;
     like( $install_out, qr/^Installing /m );
@@ -214,7 +214,7 @@
 SKIP: {
     skip 'VMS install targets do not preserve $(PREFIX)', 10 if $Is_VMS;
 
-    $install_out = run("$make install PREFIX=elsewhere DESTDIR=other/");
+    $install_out = run("$make install VERBINST=1 PREFIX=elsewhere DESTDIR=other/");
     is( $?, 0, 'install with PREFIX override and DESTDIR' ) || 
         diag $install_out;
     like( $install_out, qr/^Installing /m );
